assets/javascripts/parent_group_overview.js

```diff
@@ -0,0 +1,85 @@
+/* jshint multistr: true */
+
+function setupParentGroupOverviewAssets(group_id) {
+    var cookie_product_name = '_product_' + group_id + '_grouped_by';
+    var cookie_all_products_name = 'product_all_grouped_by';
+
+    function setCookie(name, value){
+      document.cookie = name + "=" + value;
+    }
+
+    function getCookie(name) {
+      var value = "; " + document.cookie;
+      var parts = value.split("; " + name + "=");
+      if (parts.length == 2) return parts.pop().split(";").shift();
+    }
+
+    function update_grouped_by_default_links(){
+        if (window.location.hash !== getCookie(cookie_product_name))
+            $('#grouped_by_default').show();
+        else
+            $('#grouped_by_default').hide();
+
+        if (window.location.hash !== getCookie(cookie_all_products_name))
+            $('#grouped_by_default_all').show();
+        else
+            $('#grouped_by_default_all').hide();
+    }
+
+    function update_grouped_by_classes(){
+        function removeClasses(id, classes){
+            classes.forEach(function(item){
+                $(id).removeClass(item);
+            });
+        }
+
+        var  classesToRemove = ['active', 'show', 'parent_group_overview_grouping_active'];
+
+        if (window.location.hash === '#grouped_by_build'){
+            removeClasses('#grouped_by_group', classesToRemove);
+            removeClasses('#grouped_by_group_tab', classesToRemove);
+            $('#grouped_by_build').addClass('active show');
+            $('#grouped_by_build_tab').addClass('parent_group_overview_grouping_active active');
+        }else{
+            removeClasses('#grouped_by_build', classesToRemove);
```

Comment:
Without jQuery and your helper the following should be possible: `document.getElementById('grouped_by_build').classList.remove(...classesToRemove);`

Reply:
Ah, understood. Total jquery replacement when is it possible

Reply:
Done, replaced jquery code